app/react/src/server/config/webpack.config.prod.js

```diff
@@ -47,21 +46,8 @@ export default function(configDir) {
         },
         template: require.resolve('../iframe.html.ejs'),
       }),
+      // new InterpolateHtmlPlugin(process.env),
       new webpack.DefinePlugin(loadEnv({ production: true })),
-      new UglifyJsPlugin({
-        parallel: true,
-        uglifyOptions: {
-          ie8: false,
-          mangle: false,
-          warnings: false,
-          compress: {
-            keep_fnames: true,
-          },
-          output: {
-            comments: false,
-          },
-        },
```

Comment:
@ndelangen should we replicate those settings in `optimization.minimizer` option, or are we OK with webpack [defaults](https://github.com/webpack/webpack/blob/4f399322ad1922ae7bafdeea1c4d83bf9c9851a7/lib/WebpackOptionsDefaulter.js#L271-L276)?

Reply:
Let's try to use their defaults!